openedx/features/student_account/views.py

```diff
@@ -1,33 +0,0 @@
-""" Views for a student's account information. """
-from django.http import HttpRequest
-from django.core.urlresolvers import resolve
-
-
-def local_server_get(url, session):
```

Comment:
Are you sure this function is not being used anywhere?

Reply:
Yes, we are